tests/test_pickle.py

```diff
@@ -1,7 +1,9 @@
+from future import standard_library
+standard_library.install_aliases()
 from mongorm import Document, DocumentRegistry, StringField, connect
 
 try:
-	import cPickle as pickle
+	import pickle as pickle
```

Comment:
what happens here in python2 would the `import pickle as pickle` give you cPickle or just the normal one?

Reply:
Hmmm, according to the source: https://python-future.org/_modules/future/standard_library.html it doesn't properly alias `cPickle` as `pickle` yet... even though it does the replacement 🤪 so I _think_ if we leave it as `cPickle` it's fine. Py3 should just ignore it.

Also to note, unless there's somewhere I can't see, it doesn't seem like there's anywhere else in Mongorm that actually uses `pickle`?

Reply:
I think it's to test that we can pickle a mongorm Document, which would be used in our engine code to probably cache an instance of the Document.

Reply:
Regarding your a point about `pickle` yeah I guess if we leave it and py3 ignores it then should be ok.